src/app.js

```diff
@@ -2,34 +2,140 @@ import 'foundation-sites/dist/foundation.css';
 import 'css/app.css';
 
 import $ from 'jquery';
+import _ from 'underscore';
+import Backbone from 'backbone';
 
 import Simulator from 'models/simulator';
 import QuoteList from 'collections/quote_list';
+import QuoteView from 'views/quote_view';
+import QuoteListView from 'views/quote_list_view';
+import Order from 'models/order';
+import OrderList from 'collections/order_list';
+import OrderListView from 'views/order_list_view';
+import TradeHistoryView from 'views/trade_history_view';
+
+// declaring variables, initializing lists
+
+let quoteViewTemplate;
+let tradeTemplate;
+let orderTemplate;
+let quotes;
+
+let bus = {};
+_.extend(bus, Backbone.Events);
 
 const quoteData = [
   {
     symbol: 'HUMOR',
-    price: 88.50,
+     price: 88.50,
   },
   {
     symbol: 'CLOTH',
-    price: 81.70,
+     price: 81.70,
   },
   {
     symbol: 'HABIT',
-    price: 98.00,
+     price: 98.00,
   },
   {
     symbol: 'SUPER',
-    price: 83.10,
+     price: 83.10,
   },
 ];
 
+quotes = new QuoteList(quoteData);
+const orders = new OrderList();
+
+// order form
+
+const populateForm = function populateForm () {
+  const $form_select = $('#order-form select[name="symbol"]');
+  quotes.each((quote) => {
+    const symbol = quote.get('symbol');
+    $form_select.append(`<option value="${symbol}">${symbol}</option>`);
+  });
+};
+
+const newBuy = function newBuy(e) {
+  newOrder(e, true);
+};
+
+const newSell = function newBuy(e) {
+  newOrder(e, false);
+};
+
+const newOrder = function newOrder(e, isBuy) {
+  e.preventDefault();
+  clearFormErrors();
+  const formData = getOrderFormData();
```

Comment:
You have a bunch of functions here that all deal with the order form. It might make sense to take these and wrap them in a separate view.